packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -126,20 +126,12 @@ function InteractionLayer({
   function onPointerMove(event) {
     if (creating) {
       activeTool?.handlePointerMove?.(convertEvent(event), activeMark)
-    } else {
-      // this outputs the mouse coords when not creating (ig: guideline for Polygon)
-      activeTool?.handlePointerPosition?.(convertEvent(event), activeMark)
```

Comment:
[observation] I think this else clause was only used by the Polygon tool to create the "guide line". Since the guide line is _still_ working despite the removal of handlePointerPosition(), I need to run through the code again to understand how it's working now.

Reply:
See #3389, where I removed `handlePointerPosition`, because it duplicated `handlePointerMove`.